Bots/JavaScript/Consumers/CodeFirst/WaterfallHostBot/authentication/allowedSkillsClaimsValidator.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { JwtTokenValidation, SkillValidation } = require('botframework-connector');
+const { SkillsConfiguration } = require('../skillsConfiguration');
+
+const skillsConfig = new SkillsConfiguration();
+// Load the appIds for the configured skills (we will only allow responses from skills we have configured).
+const allowedSkills = Object.values(skillsConfig.skills).map(skill => skill.appId);

```

Comment:
I see this everywhere and I don't think it's necessary to fix it here, but going forward if we are looking something up at runtime this should be a `Set`, not an array.

Reply:
Thanks @joshgummersall , we'll keep this in mind to update all the JS bots soon.